src/Acl.php

```diff
@@ -748,7 +748,7 @@ public function isAllowed($role = null, $resource = null, $privilege = null)
             // query on all privileges
             do {
                 // depth-first search on $role if it is not 'allRoles' pseudo-parent
-                if (null !== $role && null !== ($result = $this->roleDFSAllPrivileges($role, $resource, $privilege))) {
+                if (null !== $role && null !== ($result = $this->roleDFSAllPrivileges($role, $resource))) {
```

Comment:
Something too much happened here

Reply:
The `roleDFSAllPrivileges` params are only 2, and it passed 3 args on usage, so last unused arg passed can be removed

Reply:
Eh, it's `protected`, so I'm not 100% sure about this 😬

I thin it's an acceptable change though: can't expect BC on this 👍